packages/connect/src/core/AbstractMethod.ts

```diff
@@ -306,8 +306,10 @@ export abstract class AbstractMethod<Name extends CallMethodPayload['method'], P
         if (!version) return;
 
         if (
-            device.firmwareStatus === 'required' ||
-            !versionUtils.isNewerOrEqual(version, range.min)
+            this.name !== 'backupDevice' &&
+            this.name !== 'recoveryDevice' &&
```

Comment:
nit: `Array.includes`?

Reply:
:octocat:

Reply:
Ok, that's weird, let's keep it as is, then.